packages/api/src/logger.ts

```
@@ -1,8 +1,26 @@
+// Get environment variables directly to avoid circular dependency with config
+const nodeEnv = process.env.NODE_ENV || 'development';
+const logLevel = process.env.LOG_LEVEL || 'info';
+const isDevelopment = nodeEnv === 'development';
+
+// Read version from package.json at build-time for reliability
+function getPackageVersion(): string {
+  try {
+    const packageJsonPath = join(__dirname, '..', 'package.json');
+    const packageJson = JSON.parse(readFileSync(packageJsonPath, 'utf8'));
+    return packageJson.version || '1.0.0';
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)